coolwsd.xml.in

```diff
@@ -23,6 +23,7 @@
     </languagetool>
 
     <deepl desc="DeepL API settings for translation service">
+		<enabled desc="If DeepL translation settings are configured." default="false">false</enabled>
```

Comment:
there are two tabs here. It should be spaces

Reply:
also probably better to rephrase it (it sounds as if we add API url and API key the icon will appear which is not true). Maybe better to have instead "Enable Deepl translation service" and maybe add the type ( type="bool")

Reply:
Corrected the tabs and added type, thanks :) i think description is fine but..